extensions/postgres/extension.toml

```diff
@@ -1,7 +1,9 @@
 [extension]
 name = "postgres"
-version = "0.1.0"
-description = "A new extension"
+version = "0.1.1"
+description = """
+Integrate your Postgres database directly into Grafbase Gateway declaratively. This extension introspects your database schema and automatically generates a fully-functional GraphQL schema, eliminating the need to build and maintain a separate subgraph.
```

Comment:
Nitpicky, but the extension doesn't introspect (you can even define things manually). Maybe "exposes your database schema... etc."?